test/unit/spec/v2/ion/i18nTransformer_spec.js

```diff
@@ -1606,18 +1597,6 @@ describe('v2/ion/i18nTransformer', function() {
     });
   });
 
-  it('gets message key', () => {
```

Comment:
Restructured the tests into `describe` blocks to improve reporting - no tests were removed.